deploy/05_deploy_new_tenderizer_implementation.ts

```diff
@@ -0,0 +1,31 @@
+import dotenv from 'dotenv'
+import { DeployFunction } from 'hardhat-deploy/types'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+
+dotenv.config({ path: './deploy/.env' })
+const NAME = process.env.NAME || ''
+const SYMBOL = process.env.SYMBOL || ''
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  if (!NAME || !SYMBOL) {
+    throw new Error('Must provide Tenderizer Name and Symbol')
+  }
+
+  console.log('Running 05')
+  const { deployments, getNamedAccounts } = hre
+  const { deployer } = await getNamedAccounts() // Fetch named accounts from hardhat.process.env.ts
+
+  const tenderizerImplementation = await deployments.deploy(
+    NAME, {
+      from: deployer,
+      args: [],
+      log: true
+    }
+  )
```

Comment:
So this script only deploys the new impl and does not upgrade the proxy. I was simply re-running the existing deployment, it would re-use everything else, deploy the impl, and update the proxy.